crate_universe/bootstrap.bzl

```diff
@@ -1,14 +1,18 @@
 """A module for declaraing a repository for bootstrapping crate_universe"""
 
+load("@examples//third_party/openssl:openssl_repositories.bzl", "openssl_repositories")
```

Comment:
I feel strongly that we should not be crossing boundaries into examples from the top level rules. Would you be willing to add a `crate_universe/3rdparty` directory of sorts for put the definitions for OpenSSL there?

Reply:
Yeah, I'm with you - fixed!